backend/de.metas.business/src/main/java/de/metas/mforecast/impl/ForecastDAO.java

```diff
@@ -52,8 +67,41 @@ public List<I_M_ForecastLine> retrieveLinesByForecastId(final int forecastId)
 	}
 
 	@Override
-	public I_M_ForecastLine getForecastLineById (final int forecastLineRecordId)
+	public I_M_ForecastLine getForecastLineById(final int forecastLineRecordId)
```

Comment:
why do we need this method? where is used?

Reply:
It was not introduced in this commit, I see that it is used in `de.metas.material.dispo.commons.repository.CandidateRepositoryWriteService#updateCandidateRecordFromCandidate`